src/main/java/teammates/common/util/Utils.java

```diff
@@ -33,5 +35,24 @@ public static Gson getTeammatesGson() {
                 .setPrettyPrinting()
                 .create();
     }
-
+    
+    /**
+     * @param participants
+     *            a list of 'FeedbackParticipantType' objects
+     * @param delimiter
+     *            a delimiter to separate each participant (e.g. a comma sign ",")
+     * @return a string representing all participants separated by 'delimiter'
+     */
+    public static String joinParticipantTypes(List<FeedbackParticipantType> participants, String delimiter) {
```

Comment:
Perhaps you could check that `participants` is not empty and only perform the for loop and substring when it is not empty. That way we can have a single return `result` statement.